source/drivers/go.txt

```diff
@@ -0,0 +1,71 @@
+<<<<<<< HEAD
```

Comment:
I guess that you merged some conflicts.

Reply:
this should not be merged at the moment. I will revoke the PR.